website/docs/type-assertions.md

```diff
@@ -138,16 +144,11 @@ end
   → View on sorbet.run
 </a>
 
-## `T.must_because`
+Here's the same example with `T.must_because`, showing custom error messages.
+The error message is provided as a block, so that the error message is only
+computed if the exception would be raised.
```

Comment:
Nit: it's "reason" above and "error message" here; I think we should try to be consistent.